packages/feedback/src/sass/mixin/_mixin.sass

```diff
@@ -0,0 +1,285 @@
+
```

Comment:
Same as for the `_component.sass`. No need to copy the entire file for a small SPA such as this. Just take the mixins that you really need for this SPA.

Reply:
fixed it.